database/nostr-lmdb/src/store/lmdb/mod.rs

```diff
@@ -48,15 +50,21 @@ pub(crate) struct Lmdb {
 }
 
 impl Lmdb {
-    pub(super) fn new<P>(path: P, map_size: usize) -> Result<Self, Error>
+    pub(super) fn new<P>(
+        path: P,
+        map_size: usize,
+        max_readers: u32,
+        max_dbs: u32,
```

Comment:
Here I think this should be called `additional_dbs`, since 9 databases are defined internally.